memorystore/redis/server.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2018 Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START server_js]
```

Comment:
Can we get better region tags? How about, add a `memorystore_` prefix to all of them.

Reply:
Jason, I'll follow up w/ a region tag cleanup. Hope that's ok (the samples are referenced in docs to be launched soon).